app/uk/gov/hmrc/tariffclassificationfrontend/controllers/CancelRulingController.scala

```diff
@@ -45,7 +45,7 @@ class CancelRulingController @Inject()(authenticatedAction: AuthenticatedAction,
   override protected def redirect: String => Call = routes.CaseController.rulingDetails
 
   override protected def isValidCase(c: Case)(implicit request: AuthenticatedRequest[_]): Boolean = {
-    c.status == COMPLETED && c.rulingHasNotExpired
+    c.status == COMPLETED && !c.rulingHasExpired
```

Comment:
We could have a second method `c.hasLiveRuling`

Reply:
I thought to add an extra method too... But then I realised I did not want to add additional unit tests